APISpecification.yaml

```diff
@@ -235,9 +235,54 @@ paths:
                       $ref: '#/components/schemas/PopularSpecies'
         '500':
           $ref: '#/components/responses/500InternalServerError'
+  /api/metadata/genome/{genome_id}/dataset/{dataset_type}/attributes:
+    get:
+      description: Returns a list of attributes for a genome.
```

Comment:
`a list of attributes for a genome` -> `a list of attributes of a given dataset of a genome`?